src/adaptors/http1/http1_adaptor.c

```diff
@@ -53,6 +53,7 @@
      adaptor for meta-data.
  */
 
+const char *http1_alpn_protocols[HTTP1_NUM_ALPN_PROTOCOLS] = {"http/1.1", "http/1.0"};
```

Comment:
Since this commit does not implement TLS in the http1 adaptor, can we remove all http1 related files from this commit ?

Reply:
That makes sense, will do.